docs/source/tutorials/export.md

```diff
@@ -47,131 +55,35 @@ python tools/export.py configs/detection/yolox/yolox_s_8xb16_300e_coco.py \
         work_dirs/detection/yolox/epoch_300_export.pth
 ```
 
-#### Original model
-
-Eport the orginal model by setting the export config as:
-
-```shell
-export = dict(use_jit=False, export_blade=False, end2end=False)
-```
-
-#### Script model
-
-Eport the script model by setting the export config as:
-
-```shell
-export = dict(use_jit=True, export_blade=False, end2end=False)
-```
-
-#### Blade model
-
-Eport the blade model by setting the export config as:
-
-```shell
-export = dict(use_jit=True, export_blade=True, end2end=False)
-```
-
-You can choose not to save the jit model by setting use_jit=False.
-
-The blade environment must be installed successfully to export a blade model.
-
-To install the blade, you can refer to https://help.aliyun.com/document_detail/205134.html.
-
-#### End2end model
-
-Eport the model in the end2end mode by setting ''end2end=True'' in the export config:
-
+**Export configs:**
 ```shell
-export = dict(use_jit=True, export_blade=True, end2end=True)
-```
-
-You should define your own preprocess and postprocess as below (please refer to: https://pytorch.org/docs/stable/jit.html?highlight=jit#module-torch.jit ) or the default test pipeline will be used.
-
-```python
-@torch.jit.script
-def preprocess_fn(image, traget_size=(640, 640)):
-		"""Process the data input to model."""
-    pass
-
-@torch.jit.script
-def postprocess_fn(output):
```

Comment:
why remove these export guides?

Reply:
we have changed a new design, and will discuss it.